ui/app/common/domain/services/configurationService.js

```diff
@@ -183,6 +183,30 @@ angular.module('bahmni.common.domain')
             });
         };
 
+        configurationFunctions.prescriptionSMSToggle = function () {
+            return $http.get(Bahmni.Common.Constants.globalPropertyUrl, {
+                params: {
+                    property: 'sms.enablePrescriptionSMSAlert'
+                },
+                withCredentials: true,
+                transformResponse: [function (data) {
+                    return data;
+                }]
+            });
+        };
+
+        configurationFunctions.prescriptionEmailToggle = function () {
+            return $http.get(Bahmni.Common.Constants.globalPropertyUrl, {
+                params: {
+                    property: 'sms.enablePrescriptionEmailAlert'
```

Comment:
if this is an email field, why does it have the word `sms` in it?

Reply:
Does it enable some alert message, or is it for deciding if email share option will be shown/not. If so, then it should be called: `bahmni.enableEmailPrescriptionOption`